pynecone/components/datadisplay/code.py

```diff
@@ -62,18 +66,34 @@ def create(cls, *children, **props):
         # This component handles style in a special prop.
         custom_style = props.pop("custom_style", {})
 
+        if props.pop("can_copy", False):
+            code = children[0]
+            copy_button = Button.create(
```

Comment:
Maybe we can make this customizable? Like add `copy_button: pc.Component` as a prop (with this as the default) so people can set the icon/style